test/unit/org/apache/cassandra/cql3/TraceCqlTest.java

```diff
@@ -91,7 +90,7 @@ public void testCqlStatementTracing() throws Throwable
                            .enableTracing();
             TupleType tt = TupleType.of(ProtocolVersion.NEWEST_SUPPORTED, CodecRegistry.DEFAULT_INSTANCE, DataType.cint(),
                                         DataType.text(), DataType.cfloat());
-            TupleValue value = tt.newValue();
+            com.datastax.driver.core.TupleValue value = tt.newValue();
```

Comment:
nit: Maybe it would be clearer to just rename the `CQLTester` version `ObjectTupleValue` or something similar?

Reply:
I have reverted the changes in `CQLTester`.